int64 INVALID=-2

std_msgs/Header header
int64 button
int64 button_value
int64 axis
float64 axis_value